### PR TITLE
Added streamlit sidebar for candidate wage.

### PR DESCRIPTION
**Pull request by star-ranger01**

### CODE REVIEW

File: fintech_finder.py [hunk 1]
```diff
@@ -236,11 +236,11 @@ def get_people(w3):
 # Calculate total `wage` for the candidate by multiplying the candidate’s hourly
 # rate from the candidate database (`candidate_database[person][3]`) by the
 # value of the `hours` variable
-wage = hourly_rate * hours
+candidate_wage = hourly_rate * hours
```
Comment (SkyRanger05) [inline fintech_finder.py]: Change candidate_wage to wage.

**Review by SkyRanger05 — CHANGES_REQUESTED** [review]
Make change.

File: fintech_finder.py (hunk 1 above)
Comment (star-ranger01) [reply fintech_finder.py]: Ok.

**Review by star-ranger01 — COMMENTED** [review]
Made the change.

**Pull request merged**